In [3]:
import csv
import sys
csv.field_size_limit(1024 * 1024 * 1024)

131072

In [ ]:
with open('F:\\LCE\\dataset\\imdb\\csv_files\\movie_info.csv', newline='',encoding='utf-8') as csvfile:
    reader = csv.reader(csvfile)
    data_lsit = list(reader)

In [75]:
ids = []
for data in data_lsit:
    ids.append(int(data[0]))

In [76]:
max(ids),min(ids)

(2528312, 1)

###### aka_name : (901343, 1)
###### aka_title : (377960, 1)
###### cast_info : (36244344, 1)
###### char_name : (3140339, 1)
###### comp_cast_type : (4, 1)
###### company_name : (234997, 1)
###### company_type : (4, 1)
###### complete_cast : (135086, 1)
###### info_type : (113, 1)
###### keyword : (134170, 1)
###### kind_type : (7, 1)
###### link_type : (18, 1)
###### movie_companies : (2609129, 1)
###### movie_info : 
###### movie_info_idx : (1380035, 1)
###### movie_keyword : (4523930, 1)
###### movie_link : (29997, 1)
###### name : (4167491, 1)
###### person_info : (2963664, 1)
###### role_type : (12, 1)
###### title : (2528312, 1)